src/test/compile-fail/rfc1623.rs.bk

```diff
@@ -0,0 +1,98 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Umm this seems to be a rustfmt-generated "backup" file, surely that shouldn't have been committed?